test/multiFileUpload.browser.js

```diff
@@ -1,42 +1,44 @@
-import { profile, files } from './pageObjects'
+// import { profile, files } from './pageObjects'
```

Comment:
re-enable or remove this test

Reply:
I'm pretty sure we decided to `quarantine` this test. If I remember correctly, it doesn't really add much coverage and is difficult to do right, so we're skipping it for now so we can make the decision later.

Reply:
if the test isnt being used it should be removed from the codebase. we will still have it in the git history so we can bring it back. It's generally bad practice to commit unused code.